airbyte-integrations/connectors/destination-elasticsearch/src/main/java/io/airbyte/integrations/destination/elasticsearch/ConnectorConfiguration.java

```diff
@@ -48,24 +59,43 @@ public void setAuthenticationMethod(AuthenticationMethod authenticationMethod) {
 
   @Override
   public boolean equals(Object o) {
-    if (this == o)
+    if (this == o) {
       return true;
-    if (o == null || getClass() != o.getClass())
+    }
+    if (o == null || getClass() != o.getClass()) {
       return false;
+    }
+
     ConnectorConfiguration that = (ConnectorConfiguration) o;
-    return upsert == that.upsert && Objects.equals(endpoint, that.endpoint) && Objects.equals(authenticationMethod, that.authenticationMethod);
+
+    if (upsert != that.upsert) {
+      return false;
+    }
+    if (endpoint != null ? !endpoint.equals(that.endpoint) : that.endpoint != null) {
+      return false;
+    }
+    if (!Arrays.equals(certAsBytes, that.certAsBytes)) {
+      return false;
+    }
+    return authenticationMethod != null ? authenticationMethod.equals(that.authenticationMethod)
```

Comment:
How come the methods here are changed from using `Objects.equals` to these various methods? Based on this [doc of Objects.equals](https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#equals-java.lang.Object-java.lang.Object-) it seems the same logic is already applied as 


```java
authenticationMethod != null ? authenticationMethod.equals(that.authenticationMethod) : that.authenticationMethod == null;
```

wouldn't it make sense to simply have 
```java
return upsert == that.upsert && Objects.equals(endpoint, that.endpoint) && Objects.equals(authenticationMethod, that.authenticationMethod) && Arrays.equals(certAsBytes, that.certAsBytes);
```

Reply:
Updated to use Lombok instead, thanks